src/genie/libs/parser/ironware/tests/ShowIPRouteSummary/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,21 @@
+expected_output = {
+  'total': 449,
+  'protocols': {
+    'connected': 3,
+    'static': 0,
+    'rip': 0,
+    'ospf': 446,
+    'bgp': 0,
+    'isis': 0
+  },
+  'cidrs': {
```

Comment:
I understand `cidrs`, but might be better to have similar word with output for everyone.
```
`number_of_prefixes`: {
    '/24': 1,
(snip)

Reply:
I will change this to `netmask` too.

Reply:
confirmed the change.